### PR TITLE
Add cpu count to rhs table on node cpu console

### PR DESCRIPTION
**Pull request by kormat**
Also fix formatting of bounded values to be more readable.

### CODE REVIEW

**Comment by kormat** [comment]
@brian-brazil @juliusv

File: consoles/node-cpu.html [hunk 1]
```diff
@@ -1,11 +1,13 @@
 {{ template "head" . }}
 
 {{ template "prom_right_table_head" }}
-  <tr><th colspan="2">CPU</th></tr>
+  <tr>
+    <th colspan="2">CPU(s): {{ template "prom_query_drilldown" (args (printf "scalar(count(count by (cpu)(node_cpu{job='node',instance='%s'})))" .Params.instance)) }}</th>
```
Comment (brian-brazil) [inline consoles/node-cpu.html]: The colon seems a bit off for a header. CPU(n) maybe?

File: consoles/node-cpu.html [hunk 2]
```diff
@@ -1,11 +1,13 @@
...
+  </tr>
 {{ range printf "sum by (mode)(rate(node_cpu{job='node',instance='%s'}[5m])) * 100 / scalar(count(count by (cpu)(node_cpu{job='node',instance='%s'})))" .Params.instance .Params.instance | query | sortByLabel "mode" }}
   <tr>
     <td>{{ .Labels.mode | title }} CPU</td>
-    <td>{{ .Value | printf "%.3g" }}%</td>
```
Comment (brian-brazil) [inline consoles/node-cpu.html]: We should change node and node-overview too if this is changing.

File: consoles/node-cpu.html (hunk 1 above)
Comment (kormat) [inline consoles/node-cpu.html]: Doing that would make me assume the stats in the table are for a specific CPU

Comment (brian-brazil) [inline consoles/node-cpu.html]: A separate row for the count then?

Comment (kormat) [inline consoles/node-cpu.html]: I did that initially, but i didn't really like that it was the only non-percentage in the table. But that's pretty minor, so i'm happy to revert to that.

File: consoles/node-cpu.html (hunk 2 above)
Comment (kormat) [inline consoles/node-cpu.html]: Updated the PR with this done

**Pull request merged**